lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -360,13 +360,18 @@ public function getCreatePrimaryKeySQL(Index $index, $table)
      */
     protected function getCreateColumnCommentSQL($tableName, $columnName, $comment)
     {
+        $schemaSQL = 'dbo';
+        $tableSQL  = $tableName;
         if (strpos($tableName, '.') !== false) {
             [$schemaSQL, $tableSQL] = explode('.', $tableName);
-            $schemaSQL              = $this->quoteStringLiteral($schemaSQL);
-            $tableSQL               = $this->quoteStringLiteral($tableSQL);
-        } else {
-            $schemaSQL = "'dbo'";
-            $tableSQL  = $this->quoteStringLiteral($tableName);
+        }
+
+        if (strpos($tableName, '[') === false) {
```

Comment:
I think stripping the `[` `]` quotes would be better, as we can then quote it using `'` like we would normally do.